pkg/sdk/testint/sequences_integration_test.go

```diff
@@ -215,4 +215,15 @@ func TestInt_SequencesShowByID(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, id2, e2.ID())
 	})
+
+	t.Run("show by id: check fields", func(t *testing.T) {
```

Comment:
can be added to assertSequence, no need for a separate test

Reply:
Done.